erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -795,3 +796,35 @@ def update_item_quantity(source, target, source_parent):
 	doc.set_item_locations()
 
 	return doc
+
+@frappe.whitelist()
+def start_job_cards(work_order):
+	job_cards = start_and_stop_job_cards(work_order, job_started=1)
+	return job_cards
+
+@frappe.whitelist()
+def stop_job_cards(work_order):
+	job_cards = start_and_stop_job_cards(work_order, job_started=0)
+	return job_cards
+
+def start_and_stop_job_cards(work_order, job_started=None):
+	job_cards = []
+	open_job_cards = frappe.db.get_all('Job Card', {
+		'work_order':work_order, 'status':('in', ['Open', 'Work In Progress', 'Material Transferred'])
+		}, ['name'])
+	for job_card in open_job_cards:
+		job = frappe.get_doc('Job Card', job_card)
+		job.job_started = job_started
+		if job_started == 1:
```

Comment:
Just tested it out, it's letting me start a Job Card multiple times, which is causing actual worked time to multiply per line.

Check if a job is not started, and only start those job cards.